src/tests/ftest/dfuse/sparse_file.py

```diff
@@ -94,14 +94,14 @@ def test_sparsefile(self):
         fsize_write_1024thwrite = get_remote_file_size(self.hostlist_clients[0], sparse_file)
         self.log.info("File size (in bytes) after writing 1024th byte: %s", fsize_write_1024thwrite)
         # verify file did not got overwritten after dd write.
-        self.assertTrue(fsize_write_1024thwrite == self.space_before)
+        self.assertEqual(fsize_write_1024thwrite, self.space_before)
 
         # Obtain the value of 1st byte and 1024th byte in the file and
         # compare their values, they should be same.
         check_first_byte = file_obj.read(1)
         file_obj.seek(1022, 1)
         check_1024th_byte = file_obj.read(1)
-        self.assertTrue(check_first_byte == check_1024th_byte)
+        self.assertEqual(check_first_byte, check_1024th_byte)
```

Comment:
Could we add a descriptive message here that will be displayed if the assert fails?

Reply:
From a quick look, maybe
"Unexpected data in sparse file"